### PR TITLE
Apple pay example react ts

### PR DESCRIPTION
**Pull request by bdeklerk0**
Added the functionality to accept Apple Pay payments using the https://github.com/gettilled/tilled-example-monorepo/tree/master/react-ts-checkout example.

I have created 2 examples, one that uses ngrok, and the other that uses Netlify.

Please review

### CODE REVIEW

File: apple-pay-react-ts-checkout-netlify/client/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>React TS Checkout</title>
```
Comment (dpatton1992) [inline apple-pay-react-ts-checkout-netlify/client/index.html:7]: Do you want to change the title to ApplePay Checkout?

File: apple-pay-react-ts-checkout-netlify/client/public/vite.svg
Comment (dpatton1992) [inline apple-pay-react-ts-checkout-netlify/client/public/vite.svg:1]: We don't really need this if we are using the tilled logo

File: apple-pay-react-ts-checkout-netlify/client/src/assets/gym.jpg
Comment (dpatton1992) [inline apple-pay-react-ts-checkout-netlify/client/src/assets/gym.jpg:1]: We don't need these images if we aren't using subscriptions

**Review by dpatton1992 — CHANGES_REQUESTED** [review]

File: apple-pay-react-ts-checkout-netlify/client/src/assets/gym.jpg
Comment (bdeklerk0) [reply apple-pay-react-ts-checkout-netlify/client/src/assets/gym.jpg:1]: removed these now

File: apple-pay-react-ts-checkout-netlify/client/public/vite.svg
Comment (bdeklerk0) [reply apple-pay-react-ts-checkout-netlify/client/public/vite.svg:1]: Removed this now

File: apple-pay-react-ts-checkout-netlify/client/index.html (hunk 1 above)
Comment (bdeklerk0) [reply apple-pay-react-ts-checkout-netlify/client/index.html]: Change the title now.

**Review by dpatton1992 — DISMISSED** [review]
Lgtm

**Review by dpatton1992 — APPROVED** [review]

**Pull request merged**